lib/assessment.rb

```diff
@@ -20,20 +27,44 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+
+  attr_writer :name, :location
```

Comment:
that’s what I though I should do at first ) then I asked Jeff and he recommended to stick with the specs :)

Maksim
dollydasheep@gmail.com




> On Jul 16, 2015, at 9:47 PM, Matt Brendzel <notifications@github.com> wrote:
> 
> In lib/assessment.rb <https://github.com/ga-wdi-boston/ruby_objects_assessment/pull/8#discussion_r34856446>:
> 
> > @@ -20,20 +27,44 @@
> >  # Name and location should be writeable, but age should be read-only.
> >  
> >  ##
> > -# your code here
> > +class Person
> > +
> > +  attr_writer :name, :location
> I missed this earlier, but you should be using attr_accessor there, not attr_writer. attr_writer only creates a 'setter' method, not a 'getter' one. Since it's really rare that you'd want write access to a variable but not read access, attr_writer is used extremely rarely.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ga-wdi-boston/ruby_objects_assessment/pull/8/files#r34856446>.
>